packages/valory/skills/dynamic_nft_abci/behaviours.py

```diff
@@ -666,7 +668,9 @@ def update_layers(self):
             else:
                 self.context.logger.info(f"Layer {layer_name} is already up to date")
 
-    def whitelist_hash(self, image_hash: str) -> Generator[None, None, bool]:
+    def whitelist_hash(
+        self, image_hash: str
+    ) -> Generator[None, None, bool]:  # pragma: no cover
```

Comment:
Since we do not use the whitelist server, we don't call this method. Once (if) we start using it, this will need some updates. Not testing for now.